suite-native/link/src/components/Link.tsx

```diff
@@ -1,14 +1,60 @@
 import React from 'react';
-import { TouchableOpacity, TouchableOpacityProps } from 'react-native';
+import { GestureResponderEvent, TouchableOpacityProps } from 'react-native';
+import Animated, {
+    useAnimatedStyle,
+    interpolateColor,
+    withTiming,
+    useSharedValue,
+} from 'react-native-reanimated';
+
+import { useNativeStyles } from '@trezor/styles';
 
 import { useOpenLink } from '../useOpenLink';
 
 type LinkProps = TouchableOpacityProps & {
+    label: string;
     href: string;
 };
 
-export const Link = ({ href, ...touchableProps }: LinkProps) => {
+const ANIMATION_DURATION = 100;
+const IS_NOT_PRESSED_VALUE = 0;
+const IS_PRESSED_VALUE = 1;
+
+export const Link = ({ href, label }: LinkProps) => {
+    const { utils } = useNativeStyles();
     const openLink = useOpenLink();
+    const isPressed = useSharedValue(IS_NOT_PRESSED_VALUE);
+
+    const textColorStyle = useAnimatedStyle(() => ({
+        color: interpolateColor(
+            isPressed.value,
+            [IS_NOT_PRESSED_VALUE, IS_PRESSED_VALUE],
+            [utils.colors.textPrimaryDefault, utils.colors.textPrimaryPressed],
+        ),
+    }));
+
+    const handlePressIn = () => {
+        isPressed.value = withTiming(IS_PRESSED_VALUE, { duration: ANIMATION_DURATION });
+    };
+
+    const handlePress = (e: GestureResponderEvent) => {
+        openLink(href);
+        e.stopPropagation();
+    };
+
+    const handlePressOut = () => {
+        isPressed.value = withTiming(IS_NOT_PRESSED_VALUE, { duration: ANIMATION_DURATION });
+    };
 
-    return <TouchableOpacity {...touchableProps} onPress={() => openLink(href)} />;
+    return (
+        <Animated.Text
+            onPressIn={handlePressIn}
+            onPress={handlePress}
+            onPressOut={handlePressOut}
+            style={textColorStyle}
+            suppressHighlighting
+        >
+            {label}
```

Comment:
use `{children}`

Reply:
I've chose `label` because it is a prop of a `string` type. Children would suggest that the prop could be any reactnode. Since it is not a ReactNode, I belive `label` is better.